manager/service.go

```diff
@@ -52,6 +60,17 @@ func (ms *managerService) Run(ctx context.Context, c *Computation) error {
 	}
 	dur, err := time.ParseDuration(c.Timeout)
 	if err != nil {
+		detail := struct {
+			Error string `json:"error"`
+		}{
+			Error: err.Error(),
+		}
+		detailB, merr := json.Marshal(detail)
+		if merr != nil {
+			ms.logger.Warn(merr.Error())
```

Comment:
We avoid adding logger as a dependency to the svc, i.e. we separate (as much as possible) logging from the business logic.